package.json

```diff
@@ -18,21 +19,21 @@
     "act",
     "testrunner"
   ],
-  "author": "",
+  "author": "Jey <jey.nandakumar@deque.com>",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/dequelabs/axe-act-testrunner/issues"
   },
   "homepage": "https://github.com/dequelabs/axe-act-testrunner#readme",
   "dependencies": {
-    "axe-core": "^3.1.2",
+    "axe-core": "^3.1.2-canary.e67fc65",
+    "axe-reporter-earl": "0.1.0-canary.be38c0de",
     "fs-extra": "^7.0.0",
-    "markdown-table": "^1.1.2",
-    "testrunner": "git+https://github.com/auto-wcag/testrunner.git"
+    "testrunner": "git+https://github.com/auto-wcag/testrunner.git#master"
```

Comment:
It'd be nice to not rely on GitHub here. Is this package not in `npm`?

Reply:
Yup, would be nice. But this is not pushed to npm and no plans just yet.